rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+imports_granularity="Crate"
```

Comment:
Which is better: `"Module"` or `"Crate"`? This is a good point for discussion. Although the `"Module"` style is adopted by `rust-std`, but I think the `"Crate"` style is probably the better one because it avoids repeating the same crate names. All imports from the same crate is grouped together, which makes it easier for my eyes to spot the imports from the same crate.

As per the two samples above, the `"Crate"` style results in more lengthy code. But I would say the code is more readable than the one resulted from the `"Module"` style.